lib/projectassignment/project_assignment.go

```diff
@@ -62,5 +89,11 @@ func CalculateProjectDiff(oldProjects []string, newProjects []string) []string {
 		}
 	}
 
+	// if project goes from no projects to some or the reverse on update
+	// or has no projects on create
+	if (previouslyUnassigned && isUpdateRequest) || (newlyUnassigned && !isUpdateRequest) {
```

Comment:
We do have unit tests-- we *do* allow you to remove a project you have permissions on; therefore, we need to confirm that it's a create request. Otherwise, we don't permission on it.

https://github.com/chef/automate/pull/1874/files#diff-74f23555795d803f112f9f7a88c3ed2bR286